src/widget/OktaSignIn.js

```diff
@@ -153,7 +153,9 @@ var OktaSignIn = (function () {
     var authClient = createAuthClient(authParams);
 
     var Router;
-    if ((options.stateToken && !Util.isV1StateToken(options.stateToken)) || options.proxyIdxResponse) {
+    if ((options.stateToken && !Util.isV1StateToken(options.stateToken)) 
+        || options.useInteractionCodeFlow 
```

Comment:
at this moment we don't know if IDX flag is enabled or not... I think if stateToken exists, then only we can se IDX flag was enabled and hence we can use `useInteractionCodeFlow`..

Reply:
The `useInteractionCodeFlow` runs the interaction code flow, which boostraps without stateToken. Should the logic be if any of the conditions satisfied, then use V2Router to use idx?

Reply:
This issue is we need to check that IDX flag was enabled or not.. Till now we expected that if `stateToken` exists then IDX flag is enabled by default. But now it seems it's not the case.

Reply:
like a feature flag for `idx`? if there is one, only that flag should be enough here.